autoscheduler/autoscheduler/config/config.py

```diff
@@ -0,0 +1,23 @@
+import os
+import json
+
+
+def get_prop(prop):
+    """
+    get property from postgres-info.json, or create that file if needed
+    """
+    infoFile = os.path.dirname(__file__) + "/postgres-info.json"
+    try:
+        info = json.load(open(infoFile, "r"))
+        read_prop = info.get(prop)
+        if prop == "user" and read_prop == "":
```

Comment:
```suggestion
        if read_prop == "":
```
If you have it like this, then it will throw an error if user is filled but password is empty

Not sure how someone would do this, but doesn't hurt to have

Reply:
Ignore, some people can have empty passwords